### PR TITLE
OKTA-681413 Release note entries for 2024.01.0

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** New files for 2024 release notes. Add release note entries for 2024.01.0
- **Is this PR related to a Monolith release?** Yes, 2024.01.0

### Resolves:

* [OKTA-681413](https://oktainc.atlassian.net/browse/OKTA-681413)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/a1cb25499a46fe21c9caa3ea3085e901d8f87187/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d1304824-b7b2-4cb5-acfa-36f396fb2323) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/a1cb25499a46fe21c9caa3ea3085e901d8f87187/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1437fa82-1e75-4093-8274-3f022f04adf7) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/a1cb25499a46fe21c9caa3ea3085e901d8f87187/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 71 | 12 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a5433e9b-dff7-42f2-82bd-ddb9bd8f9715) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/a1cb25499a46fe21c9caa3ea3085e901d8f87187/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 79 | 11 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/340972c0-ed4f-4047-8306-9687c625d424) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.21793a68-6c6f-4fba-93ad-4b0f9534ff96?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/80fb6a355bbdd5bbf602554956d5bbdff9138f42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/df8d791a-0142-4b1f-876b-e2ceac9ed38a) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/80fb6a355bbdd5bbf602554956d5bbdff9138f42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/30129906-d549-4183-b119-d82f6de8beb4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/80fb6a355bbdd5bbf602554956d5bbdff9138f42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 73 | 39 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3f9f9d39-0869-481b-936e-629baa9e9028) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/80fb6a355bbdd5bbf602554956d5bbdff9138f42/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 72 | 21 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3a398058-5250-4a14-b5c0-b39a12f0feb6) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.65ba6f6f-674e-4b37-8274-2c1634f3a2ce?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ff19a10e70582b14653da184782a8f5c14b427c9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d83aa602-2634-47a4-97e8-5ac3f6597627) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/ff19a10e70582b14653da184782a8f5c14b427c9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/050772bc-2f83-46ce-8e1f-534012528ebc) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ff19a10e70582b14653da184782a8f5c14b427c9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 74 | 37 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/79875531-a681-4890-862c-e13cc708a586) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/ff19a10e70582b14653da184782a8f5c14b427c9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 74 | 19 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b6e7ca07-25fe-4a5a-b4d9-19ab1941e2df) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.430a12cb-6f77-4681-a828-334b1aac9978?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/eb29d7aba465e9ff444df2175608673d14b03e3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/022e77bc-131e-4419-89bf-a8b2b629f618) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/eb29d7aba465e9ff444df2175608673d14b03e3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/de0ee0e7-fa0b-48b6-88df-8cd79a255360) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/eb29d7aba465e9ff444df2175608673d14b03e3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 76 | 34 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5ab227f6-2197-41c1-9d42-69faf5030b00) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/eb29d7aba465e9ff444df2175608673d14b03e3b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 75 | 17 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3d6b225f-f625-4838-8477-750ad494836e) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.ce0f8b59-f047-4c41-b088-d87341882a0a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/0eb9d0760b9264f6b58a8610689a975c989ee363/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/28e6c318-cf51-426e-b573-f691313c0d95) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/0eb9d0760b9264f6b58a8610689a975c989ee363/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/05404b99-5f8f-417a-8098-e2522a59efd6) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/0eb9d0760b9264f6b58a8610689a975c989ee363/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 74 | 36 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13930be5-ddab-42c2-b3b2-cd4af041c836) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/0eb9d0760b9264f6b58a8610689a975c989ee363/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 75 | 17 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/460a5cdb-3aa5-431b-b8f2-1c7e31e217f5) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.45cf011a-2144-4d52-8c15-40f731e04f9f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#### New possession constraint available for Policy API is GA in Production
+
+A new `userVerification` property is available for the `constraints` object of the [Policy API](/docs/reference/api/policy/#constraints). This setting can ensure the verification of a possession factor through a PIN or biometrics. <!-- OKTA-669846 ASSURANCE_USER_VERIFICATION_POSSESSION_CONSTRAINT -->
+
+#### Read-only permission for admin role assignments is GA in Production
+
+Super admins can now assign the Roles, resources, and admin assignments permission to their delegated admins. This permission gives admins a read-only view of the admin roles, resource sets, and admin assignments in the org. See [About role permission](https://help.okta.com/okta_help.htm?type=oie&id=csh-about-role-permissions). <!-- OKTA-640563 IAM_READ_RESOURCES -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:45]:
```suggestion
Super admins can now assign the **View roles, resources, and admin assignments** permission to their delegated admins. This permission gives admins a read-only view of the admin roles, resource sets, and admin assignments in the org. See [About role permission](https://help.okta.com/okta_help.htm?type=oie&id=csh-about-role-permissions). <!-- OKTA-640563 IAM_READ_RESOURCES -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+#### Seamless ISV experience is GA in Preview
+
+Okta now provides a seamless ISV experience to optimize the [Okta Integration Network (OIN)](https://www.okta.com/integrations/) submission experience for SAML and OIDC integrations. This new experience enables independent software vendors (ISVs) to build and manually test their integration metadata before submission. This reduces the time needed for the OIN team to review and validate that the integration functions as intended, which shortens the time to publish in the OIN.
+
+This experience also incorporates communication processes in Salesforce, enabling improved collaboration internally within Okta teams and externally with ISVs. See [Publish an OIN integration](/docs/guides/submit-app-overview/) overview and [Submit an SSO integration with the OIN Wizard](/docs/guides/submit-oin-app/) guide. <!-- OKTA-663167  APP_MANIFESTS -->
+
+#### Stay signed in is EA in Preview
+
+Today, **Keep me signed in** allows the user to select whether their multifactor authenticators from previous sessions should be remembered. However, the option to select **Keep me signed in** was only available on the sign-in screen.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:55]:
```suggestion
Today, **Keep me signed in** allows the user to select whether their multifactor authenticators from previous sessions should be remembered. However, the option to select **Keep me signed in** was only available on the sign-in page.
```

File: packages/@okta/vuepress-site/docs/release-notes/2024/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| [Updated RADIUS authentication prompts](#updated-radius-authentication-prompts) | January 10, 2024 |
+
+#### DPoP support for Okta management APIs is GA in Preview
+
+You can now use OAuth 2.0 Demonstrating Proof-of-Possession (DPoP) access tokens to access Okta management APIs.
+See [Configure OAuth 2.0 Demonstrating Proof-of-Possession](/docs/guides/dpop/oktaresourceserver/main/). <!-- OKTA-673922 OKTA_RESOURCE_SERVER_DPOP_SUPPORT-->
+
+#### Read-only permission for admin role assignments is GA in Production
+
+Super admins can now assign the Roles, resources, and admin assignments permission to their delegated admins. This permission gives admins a read-only view of the admin roles, resource sets, and admin assignments in the org. See [About role permission](https://help.okta.com/okta_help.htm?type=oie&id=csh-about-role-permissions). <!-- OKTA-640563 IAM_READ_RESOURCES -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024/index.md:26]: same edit as above

**Review by susanharper-okta — APPROVED** [review]
approved with an edit.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8440d12ce293b2be6c666e5310c2d9f480c99b6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0bc31d2e-35da-47df-9028-491b11618bcd) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/8440d12ce293b2be6c666e5310c2d9f480c99b6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13c49eb1-fe8f-48be-aca3-15053ffab898) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8440d12ce293b2be6c666e5310c2d9f480c99b6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 75 | 34 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6f710a09-a59c-4fca-85a8-a37ae8c4268f) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/8440d12ce293b2be6c666e5310c2d9f480c99b6f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 75 | 17 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1ab3edc6-c3eb-4771-9786-00956117151c) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0f0d0b97-723c-4b00-9876-3b0aca207bc1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/5a62dcf6df58de6655046ee61f24a3c46becacc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1c0bf0a8-c13a-4463-8024-218645539ba9) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/5a62dcf6df58de6655046ee61f24a3c46becacc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ac7df1fa-1c4f-40f2-855a-1b00eb2ebb46) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/5a62dcf6df58de6655046ee61f24a3c46becacc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 75 | 35 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b840c965-b959-4c7c-af9a-be9f5c93df0c) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/5a62dcf6df58de6655046ee61f24a3c46becacc7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 75 | 17 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/93cd3188-48bf-46e2-b72c-66c8fde64bee) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.88e730c0-32f3-425a-a9e6-17f532494c31?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8dd61c57ebe742c69d413eb2ab62aff513432a40/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 528 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cf607607-2fbd-4ad1-bb65-d5bbf2c852b0) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/8dd61c57ebe742c69d413eb2ab62aff513432a40/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 74 | 368 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/69a747c7-eec9-4643-a684-375f51afed3e) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/8dd61c57ebe742c69d413eb2ab62aff513432a40/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024-okta-identity-engine%2Findex.md) | 75 | 34 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/622dd213-8d24-4574-bcac-5db5b0f04de1) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2024/index.md](https://github.com/okta/okta-developer-docs/blob/8dd61c57ebe742c69d413eb2ab62aff513432a40/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2024%2Findex.md) | 77 | 16 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/92c507c0-43ea-4eb5-b967-807565f57fa5) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.db4f0e09-edc3-4b36-ac36-7568579c6c3e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**